app/views/articles/show.html.slim

```diff
@@ -1,3 +1,6 @@
 h1 #{article.title}
 p
   | #{article.content}
+
+- if current_user&.administrative_role?
```

Comment:
What do you think about moving this check to policy? And check if user is authorized to publish the article in the controller before executing create method there?